portfolio/src/main/webapp/hobbies.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="UTF-8">
+    <title>Hobbies</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="script.js"></script>
+  </head>
+  <body>
+      <div class="content">
+          <h1> Hobbies </h1>
+          <nav>
+            <ul id="navigation">
+                <li><a href="index.html"> Home </a></li>
+                <li><a href="projects.html"> Projects </a></li>
+                <li><a href="hobbies.html"> Hobbies </a></li>
+            </ul>
+          </nav>
+          <div class="alt1">
```

Comment:
I couldn't (for reasons unknown to me) couldn't comment on the CSS class names in the CSS file.

I don't love `alt1` and `alt2` as class names. They're not very descriptive. You're doing this for alternating background colors for your rows (yay!). I'm not quite sure what the right name is here, but something that references the fact that this is a row background class.

Reply:
Actually, I picked this up from a comment I saw from Daniel elsewhere.

You can use one class on each `<div>` for a hobby (`hobby-item`?). In CSS:

```
.hobby-item:nth-of-type(odd) {
  background-color: #0000ff;
}

.hobby-item:nth-of-type(even) {
  background-color: #00ff00;
}
```